packages/elements/src/appstate-bar/index.ts

```diff
@@ -46,7 +46,7 @@ export class AppstateBar extends BasicElement {
   public heading = '';
 
   /**
-   * (optional) Type of state bar. Supported value are `info`, `highlight`.
+   * Type of state bar. Supported value are `info`, `highlight`.
```

Comment:
Supported values are already mentioned in type column.

```suggestion
   * Type of state bar.
```